compliance_checker/cfutil.py

```diff
@@ -296,6 +299,26 @@ def get_auxiliary_coordinate_variables(ds):
     return ret_val
 
 
+def get_forecast_metadata_variables(ds):
+    '''
+    Returns a list of variables that represent forecast reference time
+    metadata.
+
+    :param netCDF4.Dataset ds: An open netCDF4 Dataset.
+    :rtype: list
+    '''
+    forecast_metadata_standard_names = [
+        'forecast_period',
+        'forecast_reference_time',
+    ]
+    forecast_metadata_variables = []
```

Comment:
If you're only doing a simple membership check, please use a set instead of a list due to `O(1)` vs `O(n)` performance when using `in`.

Reply:
:heavy_check_mark: